CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 * Improve performance of client reset with automatic recovery and converting top-level tables into embedded tables. (Core upgrade)
 * Flexible sync will now wait for the server to have sent all pending history after a bootstrap before marking a subscription as Complete. (Core upgrade)
 * Slightly improve performance of `Realm.RemoveAll()` which removes all objects from an open Realm database. (Issue [#2233](https://github.com/realm/realm-dotnet/issues/2194))
+* Improve error messages when setting a BaseFilePath for realm or app configuration
```

Comment:
This is actually the error message when not setting the `BaseFilePath` - if you set it, you wouldn't have gotten that exception. Also, we need a link to the issue, similarly to the other changelog entries.